lib/openstax/aws/stack.rb

```diff
@@ -65,9 +65,14 @@ def template
       end
     end
 
-    def create(params: {}, wait: false)
+    def create(params: {}, wait: false, skip_if_exists: false)
       logger.info("**** DRY RUN ****") if dry_run
 
+      if skip_if_exists and exists?
```

Comment:
why "and" vs "&&"?  the ruby convention is to use && for boolean expressions like this. Using and can often lead to order precedence issues that can be difficult to debut.

Reply:
Yeah, same - python habits.